application/src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/RaptorTransferIndex.java

```diff
@@ -41,6 +43,7 @@ public static RaptorTransferIndex create(
       var transfers = transfersByStopIndex
         .get(fromStop)
         .stream()
+        .filter(transfer -> transfer.getModes().contains(mode))
```

Comment:
This is run only once and then cached for the specific combination of request parameters, correct?

Reply:
Yes, this is run once for each cache request. You can see the code that uses this in `RaptorRequestTransferCache`